ocamldoc/odoc_html.ml

```diff
@@ -2618,15 +2618,17 @@ class html =
       try
         let chanout = open_out (Filename.concat !Global.target_dir self#index) in
         let b = new_buf () in
-        let title = match !Global.title with None -> "" | Some t -> self#escape t in
         bs b doctype ;
         bs b "<html>\n";
         self#print_header b self#title;
         bs b "<body>\n";
 
-        bs b "<h1>";
-        bs b title;
-        bs b "</h1>\n" ;
+        let () = match !Global.title with
+          | None -> ()
+          | Some t ->
+              let () = bs b "<h1>" in
+              let () = bs b (self#escape t) in
+              bs b "</h1>\n" in
```

Comment:
Thanks for your editing! I can sympathize with your stylistic preference, but I would rather avoid mixing local style change with semantic changes.